GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -202,11 +204,24 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            QuickDeployConfig.Config.apkFileName = EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName,
-                GUILayout.MinWidth(FieldMinWidth));
+            QuickDeployConfig.Config.apkFileName =
+                EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName, GUILayout.MinWidth(FieldMinWidth));
+            if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
+            {
+                var defaultApkFileName = Path.GetFullPath("base.apk");
+                QuickDeployConfig.Config.apkFileName = EditorUtility.SaveFilePanel("Choose file name and location",
+                    Path.GetDirectoryName(defaultApkFileName),
```

Comment:
I think this can be simplified, but you'll have to do a small amount of research first:

What does it do if you put null? What about empty string? What about Path.GetCurrentDirectory() ?

Prefer the simplest option here that works.

Reply:
### Resolved.
Wasn't getting what you meant before